mcs/class/referencesource/mscorlib/system/missingmethodexception.cs

```diff
@@ -48,20 +48,24 @@ public override String Message
                 if (ClassName == null) {
                     return base.Message;
                 } else {
-                    // do any desired fixups to classname here.
-                    return Environment.GetResourceString("MissingMethod_Name",
-                                                                       ClassName + "." + MemberName +
-                                                                       (Signature != null ? " " + FormatSignature(Signature) : ""));
+					string res = ClassName + "." + MemberName;
+					if (!string.IsNullOrEmpty(signature))
+						res = string.Format (signature, res);
```

Comment:
This is culture-specific call which should be avoided

Reply:
What's the culture oblivious alternative for string interpolation? The string comes from the runtime.

Reply:
either don't use it or at least use string.Format with CultureInfo.InvariantCulture